docs/Scaling.md

```diff
@@ -0,0 +1,59 @@
+
+Cromwell can be thought of as performing three fundamental roles:
+
+1. Front end - handling all REST requests
+1. Runner - picking up and running workflows
+1. Summarizer - summarizing metadata
+
+In the simplest Cromwell deployment, a single Cromwell instance performs all three of these roles.
+But it is also possible to create Cromwell deployments with many Cromwell instances, gaining advantages in
+resiliency and scalability. There is only one restriction on roles: there must be exactly one Cromwell
+instance performing the role of summarizer. Apart from this restriction, any instance in a multi-Cromwell
+deployment can perform one or more of these roles. As a real world example, the current production Terra
+deployment uses 7 Cromwell instances, each dedicated to a single role: 3 front ends, 3 runners, and one summarizer.
+
+These roles are not explicit in Cromwell configuration, rather they are implied by configuration settings as
```

Comment:
TOL: Might be cool to make them explicit as a way to avoid 🤦‍♂️ config errors in the future...

Reply:
that would be a whole different and currently nonexistent ticket